staticmodules/scrapetable.py

```diff
@@ -77,7 +77,7 @@ def render(table, params, *, fetch_result):
     tablenum: int = params["tablenum"] - 1  # 1-based for user
 
     if tablenum < 0:
-        return ProcessResult(error="Table number must be at least 1")
+        return ProcessResult.coerce("Table number must be at least 1")
```

Comment:
Don't act upon this comment in this pull request (please!) ... but I'd like to illustrate more about our architecture, to be as clear as possible.

This should be `return "Table number must be at least 1"`

If we're calling `ProcessResult.coerce()`, we're doing it wrong. Modules shouldn't be aware of this type.

I went through a phase a year ago in which I thought all modules would use these "pandas types" natively. But we never wired it up for external modules, and I've since learned that Pandas is incapable of doing what we want. That's why we use Arrow now.

We always had a problem with unit tests, and that's why I wavered so much. The problem with making modules return native types was, we didn't have a good way to test the results. A `render()` function can `return None`, `return pd.DataFrame()` or `return pd.DataFrame(index=pd.RangeIndex(0))` and the outcome will be identical; but in our _unit tests_, we had to get the right one. We ended up writing unit tests that catered to the module _code_, instead of the module _function_. Our expected values often wouldn't validate, and we wouldn't catch the error until deploy. That's why I took us down the `ProcessResult()` route (before finally aborting).

Lately I've solved the unit-testing problem (for now) with `cjwkernel.tests.pandas.util.assert_process_result_equal(actual, expected)`. It coerces both `actual` and `expected`.

In a future commit, we'll wipe all reference to `ProcessResult` from staticmodules. This may be all we need to do to convert them to external modules.